src/ykfde-enroll

```diff
@@ -77,24 +77,23 @@ fi
 _tmp="$(ykinfo -"$YKFDE_CHALLENGE_SLOT" 2>&1)";
 [ "$DBG" ] && echo "$_tmp"
 
-echo "   Don't forget to press its button if necessary..."
-[ "$DBG" ] && echo "   Running NOW: ykchalresp -$YKFDE_CHALLENGE_SLOT \"$YKFDE_CHALLENGE\"..."
+echo "   Remember to touch the device if necessary."
+[ "$DBG" ] && echo "   Running: ykchalresp -$YKFDE_CHALLENGE_SLOT \"$YKFDE_CHALLENGE\"..."
 _passphrase="$(ykchalresp -"$YKFDE_CHALLENGE_SLOT" "$YKFDE_CHALLENGE" 2>/dev/null| tr -d '\n')"
 
 if [ -z "$_passphrase" ]; then
-    printf "\n   NO response from YubiKey!? - We try again! (so press the button now?!)...\n"
+    printf "\n   Yubikey did not provide a response - Lets try again!\n"
     _passphrase="$(ykchalresp -"$YKFDE_CHALLENGE_SLOT" "$YKFDE_CHALLENGE" 2>/dev/null| tr -d '\n')"
 fi
 
-[ "$DBG" ] && printf "\n   Got as response: '$_passphrase'\n"
+[ "$DBG" ] && printf "\n   Received response: '$_passphrase'\n"
 
 if [ -n "$_passphrase" ] && [ "$YKFDE_CHALLENGE_PASSWORD_NEEDED" = "1" ]; then
     _passphrase="$_pw$_passphrase"
 fi
 
 [ "$DBG" ] && echo " > Using '$_passphrase' with cryptsetup!"
-echo "You may now be prompted for an existing passphrase. This is NOT the passphrase you just entered, this is the passphrase that you currently use to unlock your LUKS encrypted drive."
-printf "   Enter password: "; if [ "$DBG" ]; then read -r OLD; else read -r -s OLD; fi
+echo "If prompted for a passphrase, enter the old registered passphrase for your LUKS container."
```

Comment:
You removed password prompt :smile:

Reply:
Yeah, I was conflicted on that change.
initially I figured the stand-alone prompt would be enough, but I only learned later on that the script relied mostly on the combined prompt.